test/unit/job_template_test.rb

```diff
@@ -129,6 +134,11 @@
       template_with_input_sets.template_inputs_with_foreign.map(&:name).must_equal ['service_name']
     end
 
+    it 'imports feature' do
+      remote_execution_feature.reload
+      remote_execution_feature.job_template.must_equal template
```

Comment:
For the test failure - the template is loaded lazily, so you need to reference it somewhere before checking the `job_templates` in the feature.